src/components/Header/SideNav/outerItemStyles.js

```diff
@@ -0,0 +1,18 @@
+import { css } from 'styled-components'
```

Comment:
This file still seems strange being called `outerItemStyles.js` while it's being used by `OuterAnchorItem`, `Dropdown` and `InnerAnchorItem`... Can it be called `sidenavItemStyles` or smthing like that?

Reply:
Good point. `fontSizeAndWeight` doesn't really belong in this file, as it's used on "inner" items as well.

I've moved `fontSizeAndWeight` into the same file that contains `itemSpacing` (margin + padding) & renamed that file `sideNavItemStyles.js`.

See what you think

Reply:
have you commited these changes yet?

Reply:
Oops. Hadn't pushed